skbio/alignment/_alignment.py

```diff
@@ -1405,6 +1408,84 @@ def sequence_length(self):
         else:
             return len(self._data[0])
 
+    def weblogo(self, reverse=False):
+        """Plots alignment as a weblogo.
+
```

Comment:
@Kleptobismol, would it be possible to add in a `Notes` section that gives a little more detail on what a weblogo is? It may be useful to also include a `References` section that links out to either [here](http://weblogo.berkeley.edu/) or the papers described in the link